source/pt-br/api/index.md

```diff
@@ -17,42 +17,42 @@ hexo.init().then(function(){
 });
 ```
 
-Option | Description | Default
+Opção | Descrição | Padrão
 --- | --- | ---
-`debug` | Enable debug mode. Display debug messages in the terminal and save `debug.log` in the root directory. | `false`
-`safe` | Enable safe mode. Don't load any plugins. | `false`
-`silent` | Enable silent mode. Don't display any messages in the terminal. | `false`
-`config` | Specify the path of the configuration file. | `_config.yml`
+`debug` | Habilita o modo debug. Mostra as messagens de debug no terminal e cria o arquivo `debug.log` no diretório raiz. | `false`
+`safe` | Habilita o modo seguro. Não carrega nenhum plugin. | `false`
+`silent` | Habilita o modo silencioso. Não mostra nenhuma mensagem no terminal. | `false`
+`config` | Especifica o caminho do arquivo de configuração. | `_config.yml`
 
-## Load Files
+## Carregar Arquivos
 
-Hexo provides two methods for loading files: `load` and `watch`. `load` is used for loading all files in the `source` folder as well as the theme data. `watch` does the same things `load` does, but will also start watching for file changes continuously.
+Hexo fornece dois métodos para carregar arquivos: `load` e `watch`. O método `load` é usado para carregador todos os arquivos da pasta `source` como também os dados do tema. O método `watch` faz a mesma coisa que o `load`, mas também observará mudanças nos arquivos continuamente..
```

Comment:
Adicionar o artigo "**O**" no início do parágrafo: "_O Hexo fornece..._".

Aguardar a definição da issue #35 para a tradução da palavra Folder.